pilot/cmd/istioctl/inject.go

```diff
@@ -18,32 +18,64 @@ import (
 	"errors"
 	"fmt"
 	"io"
+	"io/ioutil"
 	"os"
+
 	// TODO(nmittler): Remove this
```

Comment:
move down 1 line (comment applies to glog not yaml)

Reply:
@nmittler, is this `glog` import still needed or can we remove?

Reply:
I think should be able to remove, but I don't recall exactly.  It's certainly worth trying.